core/db/migrate/20210122110141_add_name_to_spree_addresses.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddNameToSpreeAddresses < ActiveRecord::Migration[5.2]
+  def up
+    add_column :spree_addresses, :name, :string
+    add_index :spree_addresses, :name
+  end
+
+  def down
+    remove_index :spree_addresses, :name
+    remove_column :spree_addresses, :name
+  end
+end
```

Comment:
Does it still count as a patch update if I need to run `bin/rails railties:install:migrations`?

Reply:
Hey! Yes, unfortunately we had to do this change at a patch level because it is fixing the migration process for switching to the new addresses fields schema. We decided to consider this a patch instead of a new feature for this reason. Theoretically, a new migration doesn't always represent a new feature, although it's strange to have. We are sorry if this is causing issues and we are open to improving the documentation to help users upgrading. Let us know your thoughts.

Reply:
I think having a more detailed [CHANGELOG](https://github.com/solidusio/solidus/blob/master/CHANGELOG.md#solidus-2110-2020-10-23) could have helped me to find the issue.

Reply:
You are right. I'm preparing a PR that updates the Changelog [here](https://github.com/solidusio/solidus/pull/3975). Anyway, you can find details of each release in https://github.com/solidusio/solidus/releases.